dev/com.ibm.websphere.appserver.features/visibility/private/explore-1.0/com.ibm.websphere.appserver.adminCenter.tool.explore-1.0.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Same comment as for the auto feature above. This is a private feature so it doesn't need translations. If the bundles need translated files then those NLS files would go in the bundle.

Reply:
Same comment here.

Reply:
We could combine the private features into the public one and not have extra nls files and just have all the icons in the public feature for the private feature.

Reply:
That's what openidConnectServer-1.0 right? It still looks odd to me but at least there is precedence

Reply:
Yeah I am guessing that these nls files are not needed because only public features need a feature description, right?